### ISSUE TITLE
Implement Material Design

### ISSUE THREAD

**Issue by bergold** [github/android#623]
I really like to use this app on my smartphone but the design needs really a redesign ;)

**Comment by mustafa01ali** [comment]
I'd love to contribute as well. However as we all agree, it's not something small and will take a good amount of effort & collaboration. Is there a plan/owner for this so that we can all be on the same page, work on it in parallel and not duplicate efforts?

**Comment by hzsweers** [comment]
^^

**Comment by fadils** [comment]
@mustafa01ali  to prevent duplication efforts, I can think of breaking of which material design you would like to create, and then make it as a specific issue. 

Perhaps, @hzsweers can be in charge and creating these issues. And then, for anyone who would like to create that feature can comment "Hey I'm on this" then I can assign the issue to this someone.

What do you guys think?

**Comment by hzsweers** [comment]
That sounds like a good plan. It would probably be helpful to add a "Material" label as well.

How open are you guys to the addition/removal of libraries? Switching to appcompat from ABS would be a big first step, but there's also a few other libraries out there that help with implementing MD.

**Comment by mustafa01ali** [comment]
@fadils That's a good idea. 

Before we jump in and start writing code, it would be better if we have an agreed design in place so that we know what the app will look like when we are done. It will also make it easier to identify the issues that need to be created. Do you guys know any designer who can help out with this?

**Comment by fadils** [comment]
@hzsweers "material" label is created. And yes, we're quite open to any libraries, as long as it's good, easily integrated and testable.

**Comment by fadils** [comment]
@mustafa01ali just follow the Google's guide. Any other thing such as colour-related problems, etc, then we can discuss it during the PR/code review.

**Comment by fadils** [comment]
All in all, we're trying to get updated. And implementing material design is on the plan.
Closing this issue now.

Any other issues related with MD, then you guys can open a new issue preferably with more details and rather specific. i.e "Implement cards for list view", "Replace ABS with appcompat". Something along this line.  A "Material" label will be tagged to make it easier for tracking MD-related issues or feature creation.
